CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+
+## Wazuh Docker v3.11.4_7.6.1
```

Comment:
This has to point `v3.12.0`

Reply:
Fixed for all tags in https://github.com/wazuh/wazuh-docker/pull/316/commits/cb60fc8b771a04b2c3beb652c305ffd5b316687e

Greetins !